queries/graphql/highlights.scm

```diff
@@ -98,6 +111,8 @@
 
 (description) @comment
 
+(comment) @comment
```

Comment:
i've merged the other PR which handles the `(comment)` node, you can remove it :)

Reply:
Done!